agents/processor/Cargo.toml

```diff
@@ -21,8 +21,8 @@ tracing-subscriber = "0.2.15"
 rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb" }
 affix = "0.1.2"
 prometheus = "0.12"
-rusoto_s3 = "0.47.0"
-rusoto_core = "0.47.0"
+rusoto_s3 = "0.48.0"
```

Comment:
is there a reason for the bump?

Reply:
Yeah ethers is now on 0.48.0 so had to bump for consistency :)